web/src/components/LikertScaleQuestion/LikertScaleQuestion.tsx

```diff
@@ -23,7 +32,7 @@ const LikertScaleQuestion = ({
   return (
     <Stack alignItems="center">
       <Stack direction="row">
-        <Square size="20px" bg="gainsboro" data-testid="square" />
+        <Square size="20px" bg={colors.brand.gray} data-testid="square" />
```

Comment:
According to the [Customize Theme section of the Chakra Docs](https://v1.chakra-ui.com/docs/styled-system/theming/customize-theme#customizing-theme-tokens) you can simply specify the color as a string: 
```ts
<Square size="20px" bg="brand.gray" data-testid="square" />
```
This will pull the value from the theme directly without you needing to import the `color` object.